example/package.json

```diff
@@ -9,11 +9,7 @@
   "cordova": {
     "plugins": {
       "@radarlabs/cordova-plugin-radar": {},
-      "cordova-plugin-radar": {
-        "NS_LOCATION_WHEN_IN_USE_USAGE_DESCRIPTION": "This app uses your location for geofencing.",
```

Comment:
Why remove these?

Reply:
when we run `npm install && cordova build`, it automatically remove those descriptions. I think updated cordova doesn't need those description in package.json.